moved/src/state_actor.rs

```diff
@@ -596,19 +599,33 @@ where
                 U256::from(outcome.gas_used).saturating_mul(normalized_tx.tip_per_gas(base_fee)),
             );
 
-            receipts.push(TransactionWithReceipt {
-                tx_hash,
-                tx: tx.into(),
-                normalized_tx,
+            let (to, from) = match &normalized_tx {
```

Comment:
nit: Just a thought, I wonder if we can add a utility method to `NormalizedExtendedTxEnvelope` to get `(to, from)` tuple. I've seen this `match` case in multiple places.